tailscale/provider.go

```diff
@@ -31,6 +31,11 @@ func Provider(options ...ProviderOption) *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("TAILSCALE_TAILNET", nil),
 				Required:    true,
 			},
+			"baseurl": {
+				Type:        schema.TypeString,
+				DefaultFunc: schema.EnvDefaultFunc("TAILSCALE_BASEURL", nil),
+				Required:    true,
```

Comment:
I think this should be optional, and the default use `https://api.tailscale.com`